circle.yml

```diff
@@ -0,0 +1,19 @@
+machine:
+  services:
+    - docker
+
+test:
```

Comment:
We could recreate the upstream tests but I'm not sure what the point of that is.  We aren't adding code changes to this repo.